spec/requests/users/create_spec.rb

```diff
@@ -29,6 +29,12 @@
     }.to change(User, :count).by(1)
   end
 
+  it 'sends email confirmation' do
+    expect {
+      subject
+    }.to change { ActionMailer::Base.deliveries.size }.by(1)
```

Comment:
I recomend to use count, in this scenarios.
When using `size` you must be aware to not have a preload value.
`count` goes direct to the DB.

Reply:
Yeah tbh we never use `#size` or `#length`